server/src/test/java/org/apache/kafka/server/AssignmentsManagerTest.java

```diff
@@ -310,12 +313,21 @@ void testOnCompletion() throws Exception {
     }
 
     private static ClientResponse buildSuccessfulResponse(AssignReplicasToDirsRequestData request) {
+        return buildSuccessfulResponse(request, topicIdPartition -> Errors.NONE);
+    }
+
+    private static ClientResponse buildSuccessfulResponse(AssignReplicasToDirsRequestData request,
```

Comment:
nit: I don't think this is always a successful response any more. Maybe rename to `buildResponse`?